main.tf

```
@@ -0,0 +1,54 @@
+/*
+Copyright 2023 Chainguard, Inc.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+terraform {
+  required_providers {
+    apko = {
+      source = "chainguard-dev/apko"
+    }
+    cosign = {
+      source = "chainguard-dev/cosign"
+    }
+    oci = {
+      source = "chainguard-dev/oci"
+    }
+  }
+}
+
+variable "apko_config_path" {
+  description = "The path to the apko image config file."
+}
+
+variable "target_repository" {
+  description = "The docker repo into which the image and attestations should be published."
+}
+
+variable "extract_package" {
+  description = "TODO."
+}
+
+provider "apko" {
+  extra_repositories = []
+  extra_keyring      = []
+  default_archs      = []
+  extra_packages     = []
```

```suggestion
  extra_repositories = ['https://packages.wolfi.dev/os']
  extra_keyring      = ['https://packages.wolfi.dev/os/wolfi-signing.rsa.pub']
  default_archs      = ['x86_64', 'aarch64']
  extra_packages     = ['wolfi-baselayout']
```

Is it okay to make these changes, and remove the underlying config from ko's apko YAMLs? Or would you prefer that we wait for this to land as-is first? (I'm fine either way, so long as we can get this common config in here eventually)

---

im looking to land and see what breaks for now 😅

i will follow up shortly